pages/en/lb4/LoopBack-3.x.md

```diff
@@ -8,7 +8,9 @@ permalink: /doc/en/lb4/LoopBack-3.x.html
 summary:
 ---
 
-This topic will help existing users of LoopBack 3.x (or earlier versions) to understand the implication of LoopBack 4:
+For existing LoopBack 3.x (or earlier versions) users, LoopBack 4 is our next generation of the framework with a completely rewritten core foundation and significantly improved programming model. To understand the motivations, strategies, and stages for LoopBack 4, please check out [Crafting LoopBack 4](Crafting-LoopBack-4.html).
```

Comment:
This wording can be improved.  It sounds like LB 4 is the next gen of the framework _only_ for existing 3.x users.

Suggest:
```
LoopBack 4 is the next generation of the LoopBack framework, with a completely rewritten core foundation and significantly improved programming model. If you're an existing LoopBack user, read [Crafting LoopBack 4](Crafting-LoopBack-4.html) to understand the motivations, strategy, and innovations behind this exciting new version.
```